app/services/cellect_client.rb

```diff
@@ -1,92 +1,83 @@
-require 'cellect/client'
-require 'subjects/cellect_session'
+# frozen_string_literal: true
 
 module CellectClient
   ConnectionError = Class.new(StandardError)
 
-  MAX_TRIES = 1
+  CELLECT_HOST = ENV.fetch('CELLECT_HOST', 'https://cellect.zooniverse.org')
```

Comment:
I think we should probably take account of the `RAILS_ENV` here and return either the production or staging Cellect URL as needed – otherwise we would need to make sure that `CELLECT_HOST` and `RAILS_ENV` are both updated together which introduces a risk of human error.

```suggestion
  if Rails.env.production?
    DEFAULT_CELLECT_HOST = 'https://cellect.zooniverse.org'
  else
    DEFAULT_CELLECT_HOST = 'https://cellect-staging.zooniverse.org'
  end
  CELLECT_HOST = ENV.fetch('CELLECT_HOST', DEFAULT_CELLECT_HOST)
```